DrewBradley/rancid-tomatillos
# Feature/error handling with router

### What does this PR do?
- This PR adds error handling that has been refactored to use React Router.

### What does it fix?
- No bugs to fix.

### Is this a feature or a fix?
- This is a feature.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 

### How should this be tested?
- Open this file in your text editor. Try changing the fetch call inside App.js and see if an error message pops up. Then fix that fetch call, and try navigating to localhost:3000/movie/ and type in any number after movie/ to see an error message pop up, it should be displayed if it is not a 6 didgit number that exactly matches a movie ID.
